gdal/frmts/pcraster/libcsf/csf.h

```diff
@@ -40,7 +40,14 @@ typedef REAL8 CSF_VAR_TYPE;
  * (0) of the file                                
  */
 typedef UINT4 CSF_FADDR32;
-typedef long  CSF_FADDR;
+typedef
```

Comment:
This looks a bit ugly. I'd prefer
```
#ifdef _WIN32
typedef __int64 CSF_FADDR;
#else
typedef long CSF_FADDR;
#endif
```

Actually this fix shows that you would have an issue with 64-bit files and 32-bit Unix builds, since long will be 32 bit there too. At some point you should consider using the VSI*L API of GDAL to solve those issues

Reply:
I reformatted the code, and thanks for the API suggestion